rest/route/admin_spawn_hosts.go

```diff
@@ -0,0 +1,36 @@
+package route
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/gimlet"
+)
+
+func makeFetchSpawnHostUsage(sc data.Connector) gimlet.RouteHandler {
+	return &adminSpawnHostHandler{sc: sc}
+}
+
+type adminSpawnHostHandler struct {
+	sc data.Connector
+}
+
+func (h *adminSpawnHostHandler) Factory() gimlet.RouteHandler {
+	return &adminSpawnHostHandler{
+		sc: h.sc,
+	}
+}
+
+func (h *adminSpawnHostHandler) Parse(ctx context.Context, r *http.Request) error {
+	return nil
+}
+
+func (h *adminSpawnHostHandler) Run(ctx context.Context) gimlet.Responder {
+	res, err := host.AggregateSpawnhostData()
```

Comment:
is it still a thing to get db data through the rest/data connector?

Reply:
I'm sort of unclear on that. There are many places where we aren't doing that currently. I personally prefer not to since we don't need to mock the DB, so I don't see the point of the additional abstraction. @tychoish do you have an opinion?

Reply:
Ideally it should be through the data connector, since the point of that interface is to be able to separate the db from the route